packages/commonwealth/client/scripts/views/components/component_kit/new_designs/cw_tooltip.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import 'components/component_kit/new_designs/cw_tooltip.scss';
```

Comment:
please change the name of file to `CWTooltip.tsx` and `CWTooltip.scss` and keep that apprach in the upcoming PRs

Reply:
Addressed, noted.